src/agent/src/sandbox.rs

```diff
@@ -135,9 +143,9 @@ impl Sandbox {
     pub fn unset_sandbox_storage(&mut self, path: &str) -> Result<bool> {
         match self.storages.get_mut(path) {
             None => Err(anyhow!("Sandbox storage with path {} not found", path)),
-            Some(count) => {
-                *count -= 1;
-                if *count < 1 {
+            Some(obj) => {
+                obj.dec_ref()?;
+                if obj.is_zero() {
```

Comment:
`inc_ref` and `dec_ref` are only used here and in the block above. Both cases involve mutable references, so it is guaranteed that there is no aliasing.

Why do you need to make it atomic?

Reply:
I consider the situation that muti containers use the same verity device at the same time.